src/main/java/com/hm/petmaster/listener/PlayerInteractListener.java

```diff
@@ -140,6 +134,17 @@ public void onPlayerInteractEntityEvent(PlayerInteractEntityEvent event) {
 		}
 	}
 
+	@EventHandler(ignoreCancelled = true)
+    public void onPlayerDamagePet(EntityDamageByEntityEvent event){
+	    if(!((event.getDamager() instanceof Projectile || event.getDamager() instanceof Player) && event.getEntity() instanceof Tameable && ((Tameable) event.getEntity()).getOwner() != null && disablePlayerDamage))
+	        return;
+	    Entity damager = event.getDamager();
+	    if(event.getDamager() instanceof Projectile && ((Projectile) event.getDamager()).getShooter() instanceof Player)
```

Comment:
What happens if it's damage caused by a mob, for instance a skeleton? Currently I'm under the expression that it would cancel it, which is not the intended behaviour.

Reply:
When the skeleton's name is the same as the pet's owner...

Reply:
When the skeleton's name is the same as the pet's owner...  :sweat_smile: